# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: architecture/hppa64/package/*/pci-rom-big-endian.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- linux-5.17/drivers/pci/rom.c.vanilla	2022-11-03 16:48:31.351130742 +0100
+++ linux-5.17/drivers/pci/rom.c	2022-11-03 16:50:32.191126494 +0100
@@ -92,9 +92,9 @@
 		void __iomem *pds;
 		/* Standard PCI ROMs start out with these bytes 55 AA */
 		if (readw(image) != 0xAA55) {
-			pci_info(pdev, "Invalid PCI ROM header signature: expecting 0xaa55, got %#06x\n",
+			pci_info(pdev, "Invalid PCI ROM header signature: expecting 0xaa55, got %#08x\n",
 				 readw(image));
-			break;
+			//break;
 		}
 		/* get the PCI data structure and check its "PCIR" signature */
 		pds = image + readw(image + 24);
